src/domain/users/index.types.d.ts

```diff
@@ -30,6 +33,18 @@ type UserQuizQuestion = {
   completed: boolean
 }
 
+type PhoneMetadata = {
+  // from twilio
+  carrier: {
+    error_code: string // check this is the right syntax
+    mobile_country_code: string
+    mobile_network_code: string
```

Comment:
in domain layer we should keep our variable name standard, please use camel case and do the parse in repo implementation

Reply:
this require a migration to do properly. so will do in a later PR.

Reply:
This is the whole reason to have the hexagonal architecture... keep the domain / app layers clean regardless of services / repo implementations. Thats what the translation layers (aka adaptors) are for.